pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -239,12 +245,136 @@ func (adc *attachDetachController) Run(stopCh <-chan struct{}) {
 		return
 	}
 
+	err := adc.populateActualStateOfWorld()
```

Comment:
I talked to @tsmetana and yes, these new world populators will fix states of worlds that could be inconsistent due to bad event ordering.